changelog/_unreleased/2025-10-16-repeat-payment-finalize.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Repeat payment finalize
+issue: #12593
+---
+# Core
+* Added `src/Core/Migration/V6_7/Migration1760438732AddConsumedToPaymentToken.php`, which introduces a new consumed column to the payment_token table to track token usage.
+* Added `src/Core/Checkout/Payment/Cleanup/CleanupPaymentTokenTask.php`, a scheduled task that runs daily to clean up expired tokens.
+* Added the feature flag `REPEATED_PAYMENT_FINALIZE` to enable the new behavior.
+* Changed `src/Core/Checkout/Payment/Controller/PaymentController.php` to perform an early return if a token has already been consumed, preventing redundant finalizations.
+* Changed `src/Core/Checkout/Payment/Cart/Token/JWTFactoryV2.php and src/Core/Checkout/Payment/Cart/Token/TokenStruct.php` to support the new consumed logic and ensure consistency during token creation and handling.
+___
+# Upgrade Information
+## Payment Token Management
```

Comment:
all `##` headlines will be extracted standalone into the upgrade files. so make sure they work out without the context of this changelog file. 

looking at the `## Cleanup Task` and the text in the major section has no headline at all